code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -206,6 +205,12 @@
 		for(var/skin in borg_skins)
 			var/list/details = borg_skins[skin]
 			reskin_icons[skin] = image(icon = details[SKIN_ICON] || 'icons/mob/robots.dmi', icon_state = details[SKIN_ICON_STATE])
+			//SKYRAT EDIT ADDITION BEGIN - ALTBORGS
+			if (!isnull(details[SKIN_FEATURES]))
+				if (R_TRAIT_WIDE in details[SKIN_FEATURES])
+					var/image/reskin = reskin_icons[skin]
+					reskin.pixel_x -= 16
```

Comment:
The problem with cyborgs comes from pixel-shifting too. Did you make sure that, when they pixel-shift and start moving again, they won't run into the issue of being permanently off-centered again?